src/posts/xwrf/index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: 'Making WRF output data pythonic using xWRF'
+date: '2022-09-20'
+authors:
+  - name: Jon Thielen
+    github: jthielen
+  - name: Lukas Pilz
+    github: lpilz
+---
+
+![xWRF logo](https://github.com/xarray-contrib/xwrf/blob/main/docs/source/_static/xwrf_logo_bg_light.svg)
+
+In the past, when analyzing [Weather Research and Forecasting (`WRF`)](https://www.mmm.ucar.edu/weather-research-and-forecasting-model) data with python you would be restricted to using the [`wrf-python`](https://wrf-python.readthedocs.io/en/latest/) package. This is not very convenient, especially when working on very large datasets since it uses compiled Fortran code in the background essentially preventing the user from scaling to cluster-scale using utilities like [`dask`](https://www.dask.org/).
+
+Here, we present to you [`xWRF`](https://github.com/xarray-contrib/xwrf/), a lightweight interface for working with the model output in [Xarray](https://docs.xarray.dev/en/stable/). Using `xWRF` enables a seamless integration of the unique `WRF` data format into Xarray and the [Pangeo](https://pangeo.io/) software stack. It achieves this by:
+
+1. transforming `WRF` data into CF- and COMODO-compliant xarray datasets
+2. converting `WRF` units into [`pint`](https://pint.readthedocs.io/en/stable/)-friendly ones
+3. generating projection coordinates and providing a [`pyproj.CRS`](https://pyproj4.github.io/pyproj/dev/api/crs/crs.html) object
+
+In this post, we will show how `xWRF` works together with other utilities in order to post-process a large `WRF` output dataset.
+
++++
+
+## Installing `xWRF`
+
+Before we start using `xWRF`, we need to install it. We can do this by following these steps!
+
+1. Install this in your python environment using pip (`pip install xwrf`)
+2. Open up a notebook and use the imports shown below!
+
+```{code-cell} ipython3
+import holoviews as hv
+import hvplot
+import hvplot.xarray
+import xarray as xr
+import xwrf
+from distributed import Client
+from dask_jobqueue import SLURMCluster
+import pint_xarray
+import numpy as np
+
+hv.extension('bokeh')
+```
+
+## Spin up a Cluster
+
+```{code-cell} ipython3
+cluster = SLURMCluster(name='dask-cluster',
```

Comment:
Can you also mention you can use a local cluster if desired? Or provide some background on which machine your are running this on?

Reply:
I switched to a local cluster, since @jthielen wanted to run it this way. If it takes waay too long, I can also let it run on the HPC, I'd just need some hints as to how to get the rendered outputs into the blog post.